mmv1/third_party/terraform/services/netapp/resource_netapp_storage_pool_test.go.tmpl

```diff
@@ -147,6 +147,108 @@ resource "google_netapp_storage_pool" "test_pool" {
 `, context)
 }
 
+{{ if ne $.TargetVersionName `ga` -}}
+func TestAccNetappStoragePool_flexAutoTierStoragePoolCreateExample_update(t *testing.T) {
+	context := map[string]interface{}{
+		"network_name":  acctest.BootstrapSharedServiceNetworkingConnection(t, "gcnv-network-config-2", acctest.ServiceNetworkWithParentService("netapp.servicenetworking.goog")),
```

Comment:
If only network is bootstrapped, `acctest.BootstrapSharedTestNetwork` can be used.

`acctest.BootstrapSharedServiceNetworkingConnection` is used to bootstrap the service networking connection resource (`google_service_networking_connection`).

Example: https://github.com/GoogleCloudPlatform/magic-modules/blob/main/mmv1/third_party/terraform/services/gkebackup/resource_gke_backup_restore_plan_test.go#L17

Reply:
Or `acctest.BootstrapSharedServiceNetworkingConnection` is what you need?